src/elements/toniq-input/toniq-input.element.ts

```diff
@@ -134,6 +134,7 @@ export const ToniqInput = defineToniqElement({
 
         label {
             flex-grow: 1;
+            width: 100%;
```

Comment:
This change is from #43 which had requested changes here that never happened: https://github.com/Toniq-Labs/toniq-labs-design-system/pull/43/files#r958342269

Please remove this change from this PR and reopen that PR, and address the requested change on that PR.